stories/text/__snapshots__/text.stories.storyshot

```diff
@@ -0,0 +1,207 @@
+    ridiculus mus mauris vitae ultricies. In nisl nisi scelerisque eu. Suspendisse interdum consectetur libero id
+    faucibus. Ut lectus arcu bibendum at varius. Placerat vestibulum lectus mauris ultrices eros in cursus turpis massa.
+    Dolor magna eget est lorem ipsum dolor sit. Aliquam nulla facilisi cras fermentum odio eu feugiat pretium nibh.
+    Augue neque gravida in fermentum et. Proin fermentum leo vel orci porta non pulvinar.
+  </p>
+  
+
+    
+  <h3>
+    3 max lines
+  </h3>
+  
+    
+  <p
+    class="fd-text fd-text--max-lines"
+    style="-webkit-line-clamp: 3;"
+  >
+    
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+    eiusmod tempor incididunt ut labore et dolore magna aliqua. Malesuada bibendum arcu vitae elementum. Ut etiam sit
+    amet nisl. Laoreet suspendisse interdum consectetur libero id faucibus nisl. Tellus in metus vulputate eu
+    scelerisque felis imperdiet proin fermentum. Ullamcorper morbi tincidunt ornare massa eget egestas purus viverra
+    accumsan. Mauris vitae ultricies leo integer malesuada nunc vel risus. Nisi est sit amet facilisis. Eu turpis
+    egestas pretium aenean. Nunc id cursus metus aliquam eleifend. Placerat in egestas erat imperdiet. Etiam dignissim
+    diam quis enim lobortis scelerisque. A erat nam at lectus. Amet nisl purus in mollis nunc sed id. Purus semper eget
+    duis at tellus at urna.
+    Aliquet eget sit amet tellus cras adipiscing enim. Risus ultricies tristique nulla aliquet enim tortor. Varius quam
+    quisque id diam vel quam elementum pulvinar. Eget aliquet nibh praesent tristique magna. Id interdum velit laoreet
+    id donec ultrices tincidunt arcu. Suspendisse ultrices gravida dictum fusce ut placerat orci nulla. Montes nascetur
+    ridiculus mus mauris vitae ultricies. In nisl nisi scelerisque eu. Suspendisse interdum consectetur libero id
+    faucibus. Ut lectus arcu bibendum at varius. Placerat vestibulum lectus mauris ultrices eros in cursus turpis massa.
+    Dolor magna eget est lorem ipsum dolor sit. Aliquam nulla facilisi cras fermentum odio eu feugiat pretium nibh.
+    Augue neque gravida in fermentum et. Proin fermentum leo vel orci porta non pulvinar.
+  </p>
+  
+
+</section>
+`;
+
+exports[`Storyshots Components/Text Wrapping 1`] = `
+<section>
+  
+    
+  <h3>
+    No whitespace
+  </h3>
+  
+    
+  <p
+    "=""
+    class="fd-text"
+  >
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+    ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+    aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore
+    eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+    deserunt mollit anim id est laborum.
+  </p>
+  
+    
+    
+  <h3>
+    Whitespace pre-wrap
```

Comment:
At first glance, this seems incomprehensible. I know it's in the code, but how can we make it more conversational?

Pre-wrapped whitespace maybe? (for the designer audience that visits the site).

Reply:
So basically it renders whitespaces, maybe `Render whitespaces`?

Reply:
@jessmetivier just in case, I uploaded PR with your variant, but I can change it in case of need.

Reply:
Ah, okay. In that case, I would clarify it by writing:

"Wrapped text with indents and whitespace"

Reply:
And in the example above say instead of "No whitespace":

No indents and whitespace

Reply:
Maybe "Wrapping" shouldn't be the title to this example section... since this is more about "Whitespace" than anything else, and indentation is a white space in itself...

Reply:
Not just visually, but literally — whitespace is a design principle that helps guide the users' eyes. In this case, makes the text more readable! 😉

Reply:
What do you think about the plural form? I mean use "Whitespace`s`"

Reply:
Whitespace is the proper term, and it is always singular. If you want to talk about white spaces individually, it works — but you can always replace it with "whitespace" when referring to design/writing.

"There is so much whitespace on this website!"
"There are so many white spaces on this website!" (Awkward, but still technically correct)